x-pack/plugins/security_solution/server/endpoint/routes/actions/response_actions.test.ts

```diff
@@ -227,24 +231,6 @@ describe('Response actions', () => {
       expect(mockResponse.ok).toBeCalled();
     });
 
-    it('reports elasticsearch errors creating an action', async () => {
```

Comment:
This test is not relevant anymore as we use "index" method only when the `logs-endpoint` indices exist. The test for this is handled later in the "with data stream" group.